lib/localization/arb/intl_tr.arb

```diff
@@ -30,11 +30,11 @@
     }
   },
   "album": "Albüm",
-  "albumQueue": "Albüm {name}",
+  "albumQueue": "Albüm: {name}",
```

Comment:
Sorry to be picky, but please remove the `:` that you added here. It is absent in the original English string and is not needed there. Here and everywhere else wher you did this.

Reply:
It would be more understantable in Turkish. When it's translated it is a bit un-understandable, but as you want of course..

Reply:
`{name}` is a variable. In the app we graphically separate the actual name from the `Album` part.
It looks like this

![_](https://user-images.githubusercontent.com/39104740/235369797-b08270fa-13a5-4d71-b68a-0dac923c7886.jpg)

Reply:
Ok, got it